internal/app/machined/pkg/system/runner/containerd/containerd_test.go

```diff
@@ -167,15 +172,6 @@ func (suite *ContainerdSuite) TestRunTwice() {
 		suite.Assert().NoError(r.Run(MockEventSink))
 		// calling stop when Run has finished is no-op
 		suite.Assert().NoError(r.Stop())
-
```

Comment:
I removed this code because shim sockets are abstracts unix sockets now with different format, so it doesn't do anything today